app/views/content/blog/11-top-tips-for-interview-success.md

```diff
@@ -17,10 +17,11 @@ keywords:
 tags:
   - becoming a teacher
   - applications
-closing_paragraph: enriching-the-lives-of-young-people
 ---
 
-$interview$
+Caroline Downing is the Primary Programme Leader at the [Bedfordshire Schools' Training Partnership](https://bedsscitt.org.uk). She interviews applicants and assesses their potential to become successful classroom professionals.
+
+She believes the key to receiving an offer for a training place lies in effective interview preparation.
```

Comment:
@EllieNodder Is it clear that the numbered section below is written by Caroline? Would it be worth adding something like 'Here are her tips....' after this sentence?

Happy for you to disagree!